smart-assembly-scaffold/packages/contracts/script/MockSsuData.s.sol

```diff
@@ -74,16 +74,16 @@ contract MockSsuData is Script {
         owner,
         8888,
         CharacterEntityRecord({ typeId: 123, itemId: 234, volume: 100 }),
-        EntityRecordOffchainTableData({ name: "harrypotter", dappURL: "noURL", description: "." }),
+        EntityRecordOffchainTableData({ name: "mockOwnerChar", dappURL: "noURL", description: "." }),
         ""
       );
     }
 
     uint256 smartStorageUnitId = vm.envUint("SSU_ID");
     createAnchorAndOnline(smartStorageUnitId, owner);
 
-    uint256 inventoryItemIn = vm.envUint("ITEM_IN_ID");
-    uint256 inventoryItemOut = vm.envUint("ITEM_OUT_ID");
+    uint256 inventoryItemIn = 100;
+    uint256 inventoryItemOut = 200;
```

Comment:
It's better to leave these variables in `.env` rather than having it hardcoded here